### PR TITLE
Backend login module

### LINKED ISSUE #3: A user should be able to login into the system

### PR DESCRIPTION
**Pull request by lubwamabenja**
The code contains the logic required to log in the user into the system

### CODE REVIEW

**Comment by mariamiah** [comment]
Please work on fixing the linting issues

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@
 **/**/__pycache__
 .vscode/
 .pytest_cache
-src.egg-info
\ No newline at end of file
+src.egg-info
+venv/
```
Comment (mariamiah) [inline .gitignore]: Please add a new line here

File: src/models/validator.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+        lower_case = re.search(r"[a-z]", self.password)
+        upper_case = re.search(r"[A-Z]", self.password)
+        numbers = re.search(r"[0-9]", self.password)
+
+        if not self.password or not all((lower_case, upper_case, numbers))\
+                or not len(self.password) > 5:
+            return False
+        else:
+            return True
```
Comment (mariamiah) [inline src/models/validator.py]: add a newline here too

File: .gitignore (hunk 1 above)
Comment (lubwamabenja) [reply .gitignore]: Apart from the linting issues ,i needed you to help me look at the logic

**Pull request merged**